person_class.rb

```diff
@@ -22,4 +22,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(date, book)
+    rental = Rental.new(date, book, self)
+    @rentals << rental unless @rentals.includes?(rental)
```

Comment:
- [ ] Just like I stated above, I believe this line of code which tries to add the newly created rental to the @rentals array is no longer necessary as that operation has been taken care of already in the Rental class constructor.